### PR TITLE
Add support for outputting a list of homepage webmentions

### PR DESCRIPTION
**Pull request by tw2113**
Includes a template file for use wherever, as well as basic shortcode and widget support.

### CODE REVIEW

File: includes/class-webmention-homepage-widget.php [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+	 */
+	public function __construct() {
+		parent::__construct(
+			'Webmention_Homepage_Webmentions_Widget',
+			esc_html__( 'Homepage Webmentions Widget', 'webmention' ),
+			[
+				'classname'   => 'webmention_homepage_webmentions_widget',
+				'description' => esc_html__( 'A widget that allows you to display your homepage webmentions.', 'webmention' ),
+			]
+		);
```
Comment (dshanske) [inline includes/class-webmention-homepage-widget.php]: WordPress naming standards suggest a correlation between filename and classname

File: includes/functions.php [hunk 2]
```diff
@@ -357,3 +357,45 @@ function webmention_extract_urls( $content, $support_media_urls = false ) {
...
 	return array_filter( $urls );
 }
+
+/**
+ * Retrieve and display webmentions sent to the desingated homepage.
+ *
+ * @since NEXT
+ */
+function webmention_get_home_webmentions() {
+	$homepage = get_option( 'webmention_home_mentions', 0 );
```
Comment (dshanske) [inline includes/functions.php]: It should short circuit if 0 before the query

Comment (tw2113) [reply includes/functions.php]: Addressed in 1998135

File: includes/class-webmention-homepage-widget.php (hunk 1 above)
Comment (tw2113) [reply includes/class-webmention-homepage-widget.php]: Addressed in 4fb9cee

**Comment by tw2113** [comment]
Somehow, i broke things, so don't merge right now.

**Comment by tw2113** [comment]
I probably did do some copy/paste and missed that, so good catch there.

**Pull request closed without merging**